presto-main/src/main/java/com/facebook/presto/operator/window/AggregateWindowFunction.java

```diff
@@ -106,7 +106,7 @@ public static WindowFunctionSupplier supplier(Signature signature, final Interna
         return new AbstractWindowFunctionSupplier(signature, null)
         {
             @Override
-            protected WindowFunction newWindowFunction(List<Integer> inputs)
+            protected WindowFunction newWindowFunction(List<Integer> inputs, boolean ignoreNulls)
             {
```

Comment:
Add `checkArgument(ignoreNulls == false)` to ensure that no one uses `ignore nulls` for not (yet) supported functions, e.g. `first_value`.